src/components/pages/History/ComputeJobs/index.tsx

```diff
@@ -107,56 +117,76 @@ async function getAssetMetadata(
   }
 
   const result = await queryMetadata(queryDid, cancelToken)
-
   return result.results
 }
 
 export default function ComputeJobs(): ReactElement {
-  const { ocean, account, config } = useOcean()
-  const { accountId } = useWeb3()
+  const { ocean, account, config, connect } = useOcean()
+  const { accountId, networkId } = useWeb3()
+  const { chainIds } = useUserPreferences()
   const [isLoading, setIsLoading] = useState(true)
   const [jobs, setJobs] = useState<ComputeJobMetaData[]>([])
-  const { chainIds } = useUserPreferences()
-  const [result] = useQuery<ComputeOrders>({
-    query: getComputeOrders,
-    variables: {
-      user: accountId?.toLowerCase()
+
+  useEffect(() => {
+    async function initOcean() {
+      const oceanInitialConfig = getOceanConfig(networkId)
+      await connect(oceanInitialConfig)
```

Comment:
Doesn't this make the table to only show the compute jobs history for the wallet network?

Reply:
That's a compromise @mihaisc suggested so we can have a functional and showable `ComputeJobs` tab.

Reply:
Ok, I get it

Reply:
It actually doesn't, there is an issue with the architecture in ocean.js. I'll fix it later, for now it works like this.
![history](https://user-images.githubusercontent.com/26000280/125414096-08e5bee3-81a2-415b-9f23-dcb8b8279aa9.gif)